onadata/apps/api/viewsets/data_viewset.py

```diff
@@ -300,6 +303,12 @@ def destroy(self, request, *args, **kwargs):
                 after_count = self.object.submission_count(force_update=True)
                 number_of_records_deleted = initial_count - after_count
 
+                # send message
+                send_message(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No value for argument 'user' in function call](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5075103)